README.md

```diff
@@ -7,6 +7,8 @@
 ![GitHub forks](https://img.shields.io/github/forks/apache/incubator-wayang?style=for-the-badge)
 ![GitHub Repo stars](https://img.shields.io/github/stars/apache/incubator-wayang?style=for-the-badge)
 
+[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Apache%20Wayang%20enables%20cross%20platform%20data%20processing,%20star%20it%20via:%20&url=https://github.com/apache/incubator-wayang&via=apachewayang&hashtags=dataprocessing,bigdata,analytics,hybridcloud,developers)
```

Comment:
hi, I prefer to jump to the home page
```suggestion
[![Twitter](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/apachewayang)
```

Reply:
The intend is to get more awareness, developer and interested parties can star the project. I'd rather include a link to the webpage instead.

Reply:
well, just my personal suggestion, let's we try it first